packages/components/src/FormField/FormField.test.tsx

```diff
@@ -167,7 +167,7 @@ it("renders a field with error", () => {
         Enter a value that is correct
       </p>,
       <div
-        className="wrapper hasErrorMessage invalid"
+        className="wrapper invalid"
```

Comment:
Why did this change in here?

Reply:
because i removed this 
![image](https://user-images.githubusercontent.com/15986172/65074679-8c8b5d00-d952-11e9-84b1-61c62f334bcb.png)